apps/backend/src/datasources/postgres/ProposalDataSource.ts

```diff
@@ -43,15 +43,17 @@ import {
 
 const fieldMap: { [key: string]: string } = {
   finalStatus: 'final_status',
-  technicalStatuses: "technical_reviews->0->'status'",
-  technicalTimeAllocations: "technical_reviews->0->'timeAllocation'",
-  technicalReviewAssignees:
+  'technicalReviews.status': "technical_reviews->0->'status'",
+  'technicalReviews.timeAllocation': "technical_reviews->0->'timeAllocation'",
+  // NOTE: For now sorting by first name only is completly fine because the full name is constructed from frist + last

```

Comment:
I am not sure that I understand your question so good. Can you please explain it little bit?

Reply:
I was ill when I made this comment and reading it again now, it's pretty stupid. Please ignore it.

Reply:
What about if two people have the same first name?

Reply:
That's a good question. I personally think it will fallback to the other sorting criteria if present or otherwise it will sort them by proposal id in the end if they are the same. As you say would be nice maybe to fallback to lastname if they are the same but it is a bit hard to do that at the moment. It will require a bit of refactor and as part of this PR wont be feasible.

Reply:
I guess only worth changing if someone complains about it.